src/utils.rs

```diff
@@ -0,0 +1,45 @@
+use std::time::Duration;
+
+#[cfg(not(target_arch = "wasm32"))]
+pub(crate) async fn async_sleep(interval: Duration) {
```

Comment:
Where is this method used? And what is it used for?

Reply:
see comment above

Reply:
Wasm mystery!